### DISCUSSION TITLE
updates to labels and other label related discussions

### DISCUSSION THREAD

**Discussion by ReesePlews** [ISO-TC211/StandardsTracker#492] [General]
i made the following unifying editorial revisions to the labels today:

a) all dashes unified as  —  in the document names
b) changed color for withdrawn documents
c) changed color for PWI documents
d) changed color for metadata general discussions
e) added 19157-1 for the ISO 19157-1:2023 edition [ we will need to remove the "19157" label when the current issues are closed ]

topics for discussion:

1) presently we have a label "New requirement" described as:  _A requirement is a request for an organization to add or support something that they do not already_. 

Q1: is this only for requirements? perhaps we should add similar labels for "New recommendation" and "New permission"?

Q2: in the description for "New requirement" i do not understand the this phrase "_...request for and organization..._", is "for" perhaps incorrect and should be "from"?

2) presently we have a label "proposal" which does not have a description; there are two issues using this label;

Q1: is this label "proposal" clear enough or should it be changed?

Q2: a description should be added, that will help.

**Comment by PeterParslow** [comment]
1. I think "requirement" here is more general, most likely something like "a standard for points of interest". Also, I think the definition of "New requirement" was a pre-existing GitHub one: "an organization" in this case is TC 211 (the owner of the repository). Perhaps word as "Request for ISO/TC 211 to standardise something that is not already standardised" ? (assuming we can change the existing GitHub labels at all)

I don't think the difference between "proposal" and "new requirement" is clear.

**Reply by ReesePlews** [reply]
@PeterParslow yes, this is issue. i dont think "New requirement" matches "Request for ISO/TC211 to standardise something..." but i am not sure what a suitable title could be, as it cannot be so long. "Standardisation requests" (a bit long) "New topics" (would need to clarify in the description) , "Project proposals" or "New proposals" with the right description could be helpful. 

i am not sure if there is enough difference between something new we decide and "proposals" so probably we revise or delete the  "proposals" label.

**Reply by JanHjelmager** [reply]
If we cannot change the GitHub definition, there is also the option of defining our own new requirement (e.g. TC211 new requirement). I know it might be a little clumsy but could still be an option.

**Reply by PeterParslow** [reply]
"If we cannot change the GitHub definition" - just to note that we can change the description of the pre-defined labels. This could be useful if we want to retain & distinguish "New requirement" (in our sense) and "proposal". But it could also be confusing to anyone who uses other GitHub repositories where the label "New requirement" could have the default definition.

**Comment by PeterParslow** [comment]
b) 
in the case of the one issue labelled 19115 it isn't referring to the old edition, it is there as a place holder for the whole multi-part standard. We could use that orange colour for any other "whole series" issues.

**Reply by ReesePlews** [reply]
@PeterParslow i agree the colouring should be different when labeling comments that related to the multi-part series as a whole.  i am not sure if there are other such comments, perhaps we just create those new labels as needed.

**Reply by JanHjelmager** [reply]
If we start using colour coding there will be a need for a description what the different colours are indicating. This place shall perhaps be linked to the colour, if possible.

**Comment by ReesePlews** [comment]
the following label lack descriptions (editorial, enhancement, general, proposal, technical) and "New requirement" has the following description "A requirement is a request for an organization to add or support something that they do not already" 

would these revisions be suitable (add new descriptions when lacking a description, and then modify the "New requirement" as we decide here?

**editorial**
Editorial issue identified within a published standard/document. (typos, numbering, etc.)

**enhancement**
Proposal to add something new to an already published standard/document.

**general**
An overall comment or issue about an published standard/document.

**proposal**  ???
???

**technical**
A specific technical issue identified within a published standard/document?

i look forward to your comments and revisions.

**Reply by JanHjelmager** [reply]
I agree there is a need for defining these labels. I also think the definition proposed by Reese Plews are fine. 

When it come to the definition of "proposal" it can be a proposal on different levels. It can be a NWIP or it can be an idea for a complete new topic for TC211 to take on board. My idea of a definition will be to restrict proposal to NWIP and leave the other type of proposal to either CAG or AG2 to discuss before entered in the standard tracker. So the definition of proposal will be "A proposal for a new work item to be included in the TC211 program of work".
